mipdb/constants.py

```diff
@@ -5,4 +5,3 @@
 class Status:
```

Comment:
This is a very small file. We can probably move its contents to other modules and get rid of it at some point.